src/components/ha-radio.ts

```diff
@@ -1,11 +1,19 @@
 import { Radio } from "@material/mwc-radio";
```

Comment:
Should we go all fancy and import just the base and the styles? [like this](https://github.com/improv-wifi/sdk-js/blob/main/src/components/ib-button.ts)

Reply:
👍🏼 yes